poppy/utils.py

```diff
@@ -1601,32 +1604,32 @@ def sort_sptype(typestr):
 
     if "Flat" in sptype:
         if sptype == "Flat spectrum in F_nu":
-            spec = pysynphot.FlatSpectrum(1, fluxunits='fnu')
+            spec = SourceSpectrum(ConstFlux1D, amplitude=1 * syn_u.FNU)
         elif sptype == "Flat spectrum in F_lambda":
-            spec = pysynphot.FlatSpectrum(1, fluxunits='flam')
-        spec.convert('flam')
-        return spec * (1. / spec.flux.mean())
```

Comment:
I don't know why it is being normalized here. In the refactored package, flat spectrum is analytic and wavelength to evaluate it over isn't known ahead of time, so it is quite hard to normalize like this.